pkgs/tools/security/rng-tools/default.nix

```diff
@@ -25,6 +25,10 @@ stdenv.mkDerivation rec {
 
   postPatch = ''
```

Comment:
BTW the whole hook could be written like

```nix
postPatch = optionalString withPkcs11 ''
  doSomething
'';
```

and avoid this weird imbricated set of double single-quotes